lib/mwUtil.js

```diff
@@ -341,7 +341,8 @@ mwUtil.createRelativeTitleRedirect = (path, req, options) => {
     const backString = Array.apply(null, { length: pathSuffixCount }).map(() => '../').join('');
     let location;
     if (!options.dropPathAfterTitle) {
-        const pathPatternAfterTitle = path.substring(path.indexOf('{title}') - 1);
+        let pathPatternAfterTitle = path.substring(path.indexOf(`{${titleParamName}`) - 1);
+        pathPatternAfterTitle = pathPatternAfterTitle.replace(/term/, 'title');
```

Comment:
No way! This is super specific to the case that we have here right now. What if we're gonna have a different endpoint in the future where we decide to call the page title 'ugabugafoobarxxyyblombalabam' - we would need to replace that with 'title' as well.

This is a utility function, and it does take `titleParamName` as an option, so it has to work correctly  whatever the `titleParamName` was provided to it right? Now it will only work if it's 'title' or a 'term'